decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -174,7 +174,13 @@ def generate_localized_title(field = nil, skip_injection: false)
     tos_agreement { "1" }
     avatar { Decidim::Dev.test_file("avatar.jpg", "image/jpeg") }
     personal_url { Faker::Internet.url }
-    about { generate_localized_title(:user_about, skip_injection: skip_injection) }
+    about do
+      if skip_injection
+        generate(:title)
+      else
+        "<script>alert(\"user_about\");</script> #{generate(:title)}"
+      end
+    end
```

Comment:
In develop this is like this
```suggestion
    about { generate_localized_title(:user_about, skip_injection:) }
```